src/renderers/webxr/WebXRManager.js

```diff
@@ -633,6 +633,12 @@ class WebXRManager extends EventDispatcher {
 
 			if ( camera.isPerspectiveCamera ) {
 
+				if ( camera.userData.previousFov === undefined ) {
+
+					camera.userData.previousFov = camera.fov;
+
+				}
+
```

Comment:
Camera fov and zoom should be memoized at the top-level of this class and overriden/restored on `sessionstart` and `sessionend`. You will find this for renderer properties, and it would be the same here.